src/family.js

```diff
@@ -296,7 +296,7 @@ Family.prototype.get = function(options, callback) {
 
   this.getMetadata(gaxOptions, function(err, metadata) {
     if (err) {
-      if (err.code === 5 && autoCreate) {
```

Comment:
I'm seeing `err.code=404` using autoCreate. Will this sometimes be code 5 instead?

Reply:
`5` is the grpc code for 404. I'm not sure why we chose to use http sometimes, After this release there should be no more errors at all when using autoCreate (unless creation fails for some reason)